Extension/package.json

```diff
@@ -3078,6 +3078,17 @@
                         "maximum": 65536,
                         "scope": "machine"
                     },
+                    "C_Cpp.maxSymbolSearchResults": {
+                        "type": [
+                            "integer",
+                            "null"
```

Comment:
Is `null` actually ok for this setting?  I would recommend removing that.

Reply:
I copied the definition from other settings - null/unspecified means 'default'. Is that not what the other settings do?

Reply:
null does not mean default.  The default is 5000 and VS Code will return that value if the user has not set the setting.

Reply:
Yeah, I think `null` is fine/expected.

Reply:
I've removed `null`